dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/infracrl_test/master/conf/ssl/ca/infra_serials

```diff
@@ -1 +1 @@
-23
+
```

Comment:
Why is this empty now?

Reply:
infra-crl code should populate the infra_serials file using the certnames in the infra_inventory.txt. That this was set to the serial number of the old certname was an oversight in our original testing. Removing this provides better integration testing (and broke when I regenerated the certs).